x-pack/test/functional/apps/dashboard/reporting/README.md

```diff
@@ -0,0 +1,124 @@
+## The Reporting Tests
```

Comment:
This README is cut/pasted from the original README in `x-pack/test/reporting/README`. The text about running the baseline snapshot tests goes here. It looks like there may be some older text that was even obsolete in master (tests for BWC generation URLs)